projects.json

```diff
@@ -973,7 +981,15 @@
         "action": "BuildSwiftPackage",
         "build_tests": "true",
         "configuration": "release",
-        "tags": "sourcekit-disabled swiftpm"
+        "tags": "sourcekit-disabled swiftpm",
+        "xfail": [
+            {
+                "issue": "rdar://109999583",
```

Comment:
This only affects release builds I believe you'll need
```json
"configuration": "release"
```

Reply:
Good catch! Fortunately I don't think we'll need this PR after all

Reply:
Even better 👍